src/js/Controllers/UIController.js

```diff
@@ -210,9 +212,10 @@ class UIController {
                     rpc.params.cancelID,
                     rpc.params.timeout
                 ))
-                var timeout = rpc.params.timeout === 0 ? 15000 : rpc.params.timeout
+                const defaultTimeout = 15000;
+                const timeout = rpc.params.timeout === 0 ? defaultTimeout : rpc.params.timeout
                 this.endTimes[rpc.id] = Date.now() + timeout;
-                this.timers[rpc.id] = setTimeout(this.onPerformInteractionTimeout, timeout, rpc.id, rpc.params.appID)
+                this.timers[rpc.id] = setTimeout(this.onPerformInteractionTimeout, timeout - RESPONSE_CORRELATION_MS, rpc.id, rpc.params.appID) 
```

Comment:
@iCollin To prevent delay issues

Reply:
This needs to be compensated for in another place. If a mobile app sends a specific timeout, that value must be used by the HMI. I previously suggested adding a "correlation buffer" amount of MS to core's timeout instead of subtracting it from app specified values like we are here.

Reply:
@iCollin Timeout modifying has been removed 7e05879513fb94f0876fdcd916b0691b58bf8c63